ring/replication_set_test.go

```diff
@@ -241,3 +241,60 @@ func TestReplicationSet_Do(t *testing.T) {
 		})
 	}
 }
+
+func TestHasReplicationSetChangedWithoutStateIgnoresTimeStampAndState(t *testing.T) {
+	// Only testing difference to underlying Equal function
+	rs1 := ReplicationSet{
+		Instances: []InstanceDesc{
+			{Addr: "127.0.0.1"},
+			{Addr: "127.0.0.2"},
+			{Addr: "127.0.0.3"},
+		},
+	}
+	tests := map[string]struct {
+		rs                                         ReplicationSet
```

Comment:
[nit] Maybe calling it `nextState` could help to clarify it. Also naming fields in the test case. They're nit, up to you whether picking it up or not.

Reply:
fair